orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -65,6 +65,10 @@ export PGSSLMODE=verify-full
 {%- endif %}
 
 errorExit() {
```

Comment:
are we sure this function will be called on every error? I can see cases when script just dies without calling it. Maybe we need another cleanup salt to recover from the error?

Reply:
@votez Do you mind pointing me to which error this method is not handling? We only need to remove the temp files after `copyToLocal`. Sorry I looked at the code and didn't find anything that was not using that errorExit...Thank you!

Reply:
you are right - it seems like it is properly handled.